lib/ansible/config/base.yml

```diff
@@ -1314,7 +1314,7 @@ DISPLAY_SKIPPED_HOSTS:
   name: Show skipped results
   default: True
   description: "Toggle to control displaying skipped task/host entries in a task in the default callback"
-  env: [{name: DISPLAY_SKIPPED_HOSTS}]
+  env: [{name: ANSIBLE_DISPLAY_SKIPPED_HOSTS}]
```

Comment:
you can add a new variable, don't just remove the old one, it requires a deprecation period